packages/big-design/src/components/Dropdown/styled.ts

```diff
@@ -18,4 +19,12 @@ export const StyledLink = styled.a`
   }
 `;
 
+export const StyledListGroupHeader = styled(StyledGroupHeader)`
```

Comment:
I'm not sure I understand your suggestion - Do you mean update the `ListGroupHeader` component, or do you mean to do `styled(ListGroupHeader)` instead of `styled(StyledGroupHeader)`? The latter can't be done due to `className` and `style` being destructured and removed from the `ListGroupHeader`, and I didn't want to affect the `List` component by changing the original `ListGroupHeader` without direction from design. Or did you mean something else?

Reply:
I'm thinking he meant to update `StyledGroupHeader` instead of extending it and creating a new `Component`.

Reply:
Per video conversation, will be moving these styles to the `StyledGroupHeader`